### PR TITLE
Update crab position in deposit UI

### PR DESCRIPTION
**Pull request by adam5909099**
# Task:
https://github.com/opynfinance/squeeth-monorepo/issues/212

## Description
Show no crab position if the user doesn't have and a loading state if loading.
image

Fixes # (issue)
https://github.com/opynfinance/squeeth-monorepo/issues/212

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested
Go to strategies page with no crab position and check if it shows no position

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

**Comment by KMKoushik** [comment]
PnL% is not shown
Screenshot 2022-03-22 at 10 28 27 AM

**Review by KMKoushik — CHANGES_REQUESTED** [review]
I am not sure why, but PnL % is not shown in preview but working in live

File: packages/frontend/src/components/Trade/index.tsx [hunk 1]
```diff
@@ -31,6 +31,12 @@ const useStyles = makeStyles((theme) =>
       zIndex: 20,
       background: '#2A2D2E',
     },
+    displayBlock: {
+      display: 'block',
```
Comment (KMKoushik) [inline packages/frontend/src/components/Trade/index.tsx:35]: Is this used?

**Review by KMKoushik — APPROVED** [review]
Jus check if the added styles are used. LGTM

**Review by EstherSeyi — APPROVED** [review]
LGTM

**Pull request merged**